src/messages/en.json

```diff
@@ -36,6 +36,7 @@
    "github.deploy.app.complete": "Deployment of application *%s* is complete and the app has been started.",
    "github.deploy.app.unknown": "Deployment status of application *%s* is unknown at this time. Check the status of the application and logs for further information.",
    "github.deploy.error": "A deployment error occurred for app *%s*: %s.",
+	 "github.deploy.manifest.error": "Invalid manifest.yml file for app *%s*.",
```

Comment:
Minor - but could you clean up the indentation here?  If its like the issue I had with my Atom editor, you need to use spaces to indent.  I turned on a feature there where I can see the space more explicitly, which helps long term.